### ISSUE TITLE
 Dependencies just get changed and the author do nothing with it.

### ISSUE THREAD

**Issue by Lysander086** [Apress/pro-spring-boot-2#5]
I somehow not recommend this book.
Every chapter code demo can't be run normally, and it requires me to search internet for solutions.

For instance, chapter 5, ToDo App with spring jap and spring data REST.
I follow the guides, make those dependencies in the spring initializr, but just errors and need to search again.
How annoying.

**Comment by jerryperezperez** [comment]
You're absolutely right. The dependencies are outdated and many examples break out of the box, which is frustrating when you're trying to learn. 
I've been working on a modernized fork with current dependencies, migrated to Spring Boot 3 and Java 17 to ensure all the book's applications work properly. You can take a look at it here: [pro-spring-boot-2-migration](https://github.com/jerryperezperez/pro-spring-boot-2-migration) and feel free to contribute if you wish!
